builtin/pack-objects.c

```diff
@@ -4122,6 +4126,105 @@ static void mark_bitmap_preferred_tips(void)
 	}
 }
 
+static inline int is_oid_interesting(struct repository *repo,
+				     struct object_id *oid)
+{
+	struct object *o = lookup_unknown_object(repo, oid);
+	return o && !(o->flags & UNINTERESTING);
```

Comment:
The (quite misleadingly-named) [`lookup_unknown_object()` function](https://github.com/git-for-windows/git/blob/1e20af0e809e08cca5714727dfa0fb50c0c4664d/object.c#L186-L192) calls `lookup_object()` and if that returned `NULL`, calls `create_object()` and returns that.

Given the `o &&` part of the condition, I suspect that we want to use `lookup_object()` here?